scheduler/scheduler-node/src/main/java/org/ow2/proactive/scheduler/task/TaskLauncher.java

```diff
@@ -148,12 +144,18 @@ public void doTask(ExecutableContainer executableContainer, TaskResult[] previou
             progressFileReader.start(dataspaces.getScratchFolder(), taskId);
 
             TaskContext context = new TaskContext(executableContainer, initializer, previousTasksResults,
-                dataspaces.getScratchURI(), dataspaces.getInputURI(), dataspaces.getOutputURI(),
-                dataspaces.getUserURI(), dataspaces.getGlobalURI(), progressFileReader.getProgressFile()
-                        .toString(), getHostname());
+                    dataspaces.getScratchURI(), dataspaces.getInputURI(), dataspaces.getOutputURI(),
+                    dataspaces.getUserURI(), dataspaces.getGlobalURI(), progressFileReader.getProgressFile()
+                    .toString(), getHostname());
 
             File workingDir = getTaskWorkingDir(context, dataspaces);
 
+            logger.info("Task working dir : " + workingDir);
```

Comment:
The log level in the scheduler-node execution is a bit confusing. Those logs are never set to debug. They might be set this way if ever we really want to debug the indepth functioning of the TaskLaunchers. The info level is related to all information they are useful when we receive logs from a client installation. In that case, knowing what are the location of dataspaces, which file transfers occurred etc is mandatory to understand any problem which might have happened several days ago.